improver.py

```diff
@@ -0,0 +1,75 @@
+# Written by Goncalopp and RedFantom
+# Thranta Squadron GSF CombatLog Parser, Copyright (C) 2016 by Goncalopp and RedFantom
+# All additions are under the copyright of their respective authors
+# For license see LICENSE
+from PIL import ImageEnhance, Image, ImageOps
+import files
+import numpy
+import cv2
+
+
+class ImageImprover(object):
+    """
+    Improver class that provides a wrap around the ImageEnhance classes from PIL/pillow
+    The Improved image can be accessed through object.image, this is a PIL/pillow RGB
+    format image, that is NOT compatible with the cv2 based files.ImageFile class!
```

Comment:
That is probably a good idea. That would work fine with the code, and you can convert the image back to the `cv2` format upon returning it. I will change the class in this branch to a set of functions in that case.